types/link2aws/index.d.ts

```diff
@@ -0,0 +1,13 @@
+// Type definitions for link2aws 1.0
+// Project: https://github.com/link2aws/link2aws
+// Definitions by: Kit Peters <https://github.com/popefelix>
```

Comment:
This doesn't match your GitHub username

Reply:
It's ok by me. I and @kpeters-cbsi are one and the same.